src/utils/generate_pot.sh

```diff
@@ -1,47 +1,58 @@
-# but we want them as gettext file references (#: file:line)
-sed 's/^#\. \(.\+:[0-9]\+\)$/#: \1/' < messages_game.pot > messages_game.pot.tmp
-mv messages_game.pot.tmp messages_game.pot
-
-src/utils/gender_context.pl src/gamelogic/cgame/cg_event.c >> messages_game.pot
-
-find src/gamelogic/ -name '*.c' | sort | xgettext --from-code=UTF-8 -j -o messages_game.pot -k_ -kN_ -kP_:1,2 -k -f -
-
-find main/ui/ -name '*.menu' | sort | xgettext --from-code=UTF-8 -C -o messages_game.pot -j \
--kCVAR:1,4t \
--kMULTI:1,5t \
--kCOMBO:1,3t \
--kSLIDER:1,6t \
--kYESNO:1,4t \
--kMMBUTTON \
--kCVAR:2,5t \
--kCVAR_INT:2,5t \
--kMULTI:2,6t \
--kMULTIX:2 \
--kCOMBO:2,4t \
--kCOMBOX:2 \
--kSLIDER:2,7t \
--kYESNO:2,5t \
--kBUTTON:2,4t \
--kBUTTONX:2 \
--kBIND:2 \
--kBINDX:2 \
--kSLIDERX:2 \
--kTEXTX:2 \
--kVSAY_ITEM:1 \
--kYESNOX:2 \
--kSAY_ITEM:2 \
--f -
+#! /usr/bin/env bash
+
+# ===========================================================================
+#
+# Copyright (c) 2017-2020 Unvanquished Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+# ===========================================================================
+
+set -e # exit if a command fails
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+repo_dir="$(realpath "${script_dir}/../..")"
+dpk_dir="${repo_dir}/pkg/unvanquished_src.dpkdir"
+pot_dir="${dpk_dir}/translation"
+pot_file="${pot_dir}/game.pot"
+
+temp_pot_file="$(mktemp)"
+
+touch "${temp_pot_file}"
+
+cd "${dpk_dir}"
+find -type f -name '*.rml' \
+| sort \
+| xargs -I'{}' "${script_dir}/generate_rml_pot.py" '{}' \
+	>> "${temp_pot_file}"
+
+cd "${repo_dir}"
+"${script_dir}/generate_gender_pot.pl" \
+	'src/cgame/cg_event.cpp' \
+	>> "${temp_pot_file}"
+
+find 'src/cgame' 'src/sgame' 'src/shared' \
+	-type f -name '*.c' -o -name '*.cpp' \
+| sort \
+| xgettext --from-code=UTF-8 \
+	-j -o "${temp_pot_file}" \
+	-k_ -kN_ -kP_:1,2 -k -f -
```

Comment:
`G_` seems to be missing... oh well it's not a regression I guess

Reply:
I'm not sure to get it.

Reply:
Never mind, I forgot that there is a separate script for those.